### ISSUE TITLE
Running Bun Install in Docker

### ISSUE THREAD

**Issue by day-mon** [oven-sh/bun#15882]
### How can we reproduce the crash?

- Running bun install in context to a docker container. Seems like it was a post-install step that was doing some prisma generation

### Relevant log output

```shell

```

### Stack Trace (bun.report)

Bun v1.1.40-canary ([`b254e69`](<https://github.com/oven-sh/bun/tree/b254e6932279ee6ab7aa924d98d4d1b52a601bfe>)) on linux aarch64 [RunAsNodeCommand]

Segmentation fault at address 0xAAAAAAAAAAAAAAAA

- [`js_parser.zig:23467`](<https://github.com/oven-sh/bun/blob/b254e6932279ee6ab7aa924d98d4d1b52a601bfe/src/js_parser.zig#L23467>): `src.js_parser.NewParser_`
- [`js_parser.zig:4264`](<https://github.com/oven-sh/bun/blob/b254e6932279ee6ab7aa924d98d4d1b52a601bfe/src/js_parser.zig#L4264>): `src.js_parser.Parser._parse__anon_141409`
- [`js_parser.zig:3090`](<https://github.com/oven-sh/bun/blob/b254e6932279ee6ab7aa924d98d4d1b52a601bfe/src/js_parser.zig#L3090>): `src.cache.JavaScript.parse`
- [`bundler.zig:1408`](<https://github.com/oven-sh/bun/blob/b254e6932279ee6ab7aa924d98d4d1b52a601bfe/src/bundler.zig#L1408>): `src.bundler.Bundler.parseMaybeReturnFileOnlyAllowSharedBuffer__anon_121839`
- [`bundler.zig:1255`](<https://github.com/oven-sh/bun/blob/b254e6932279ee6ab7aa924d98d4d1b52a601bfe/src/bundler.zig#L1255>): `src.bundler.Bundler.parseMaybeReturnFileOnly__anon_116983`
- [`module_loader.zig:1656`](<https://github.com/oven-sh/bun/blob/b254e6932279ee6ab7aa924d98d4d1b52a601bfe/src/bun.js/module_loader.zig#L1656>): `src.bun.js.module_loader.ModuleLoader.transpileSourceCode__anon_116927`
- [`module_loader.zig:2367`](<https://github.com/oven-sh/bun/blob/b254e6932279ee6ab7aa924d98d4d1b52a601bfe/src/bun.js/module_loader.zig#L2367>): `Bun__transpileFile`
- [`ModuleLoader.cpp:609`](<https://github.com/oven-sh/bun/blob/b254e6932279ee6ab7aa924d98d4d1b52a601bfe/src/bun.js/bindings/ModuleLoader.cpp#L609>): `Bun::jsFunctionRequireCommonJS`

Features: jsc

**Comment by DonIsaac** [comment]
@day-mon If this isn't a private repo, could you share your package.json and prisma schema version with us? The more info the better.

If not, could you please tell me what version of Prisma you're using?

**Comment by zhangchi0104** [comment]
Hi

I encountered the same issue on github action ([Action Run](https://github.com/zhangchi0104/kuma-space/actions/runs/12441995445/job/34739655727))
```
#14 103.6 ============================================================
#14 103.6 Bun v1.1.41 (b8f28ed8) Linux x64 (baseline)
#14 103.6 Linux Kernel v6.8.0 | glibc v2.31
#14 103.6 CPU: sse42 popcnt avx avx2
#14 103.6 Args: "node" "scripts/postinstall.js"
#14 103.6 Features: jsc tsconfig tsconfig_paths 
#14 103.6 Builtins: "bun:main" "node:fs" "node:path" "node:string_decoder" "node:util/types" 
#14 103.6 Elapsed: 48ms | User: 41ms | Sys: 11ms
#14 103.6 RSS: 1.07GB | Peak: 1.12GB | Commit: 1.07GB | Faults: 0
#14 103.6 
#14 103.6 panic(main thread): Segmentation fault at address 0x0
#14 103.6 Crashed while visiting /app/apps/kuma-elysia/node_modules/prisma/node_modules/@prisma/engines/node_modules/@prisma/get-platform/dist/chunk-6HZWON4S.js
#14 103.6 oh no: Bun has crashed. This indicates a bug in Bun, not your code.
#14 103.6 
#14 103.6 To send a redacted crash report to Bun's team,
#14 103.6 please file a GitHub issue using the link below:
#14 103.6 
#14 103.6  https://bun.report/1.1.41/Bn1b8f28edChgg8//Dog9hlE+8mO0qz3nDig49hD2i2rhD++6wgD0l9vgDujqxgD4+4k9D_A2AA
#14 103.6 
#14 103.6 
#14 103.6 error: postinstall script from "@prisma/engines" terminated by SIGILL (Illegal instruction)
#14 141.6 Illegal instruction (core dumped)
#14 ERROR: process "/bin/sh -c bun install" did not complete successfully: exit code: 132
```

on the repo: https://github.com/zhangchi0104/kuma-space

**Comment by Jarred-Sumner** [comment]
Please upgrade to Bun v1.1.42

This was fixed in #15911
